jupyterhub/tests/test_spawner.py

```diff
@@ -103,7 +103,8 @@ def wait():
 
 
 async def test_single_user_spawner(app, request):
-    user = next(iter(app.users.values()), None)
+    orm_user = app.db.query(orm.User).first()
+    user = app.users[orm_user]
```

Comment:
This works because of https://github.com/jupyterhub/jupyterhub/blob/3e17b47ec3309b1b15e6c9963107db1981554397/jupyterhub/user.py#L73 🎉